src/app/targets/targets.controller.single.js

```diff
@@ -14,9 +14,6 @@
     $scope.targetAlias = $scope.targetId;
     $scope.stats = [];
     $scope.target = {};
-    $scope.targetData = {
-       id: '<strong>' + $routeParams.targetId + '</strong>'
-    };
 
```

Comment:
We define the mark-ups in the translation files, and we pass the variable to the translate function. I already removed all these declarations. I had forgotten this one. :wink: